api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessageProcessor.java

```diff
@@ -34,16 +40,41 @@ public void processAllMessages() {
     }
 
     private void processMessagesOfType(MessageType messageType) {
-        List<Message> allMessagesOfType = messageRepository.findAllMessagesByTypeOrderedByDate(messageType);
-        if (!CollectionUtils.isEmpty(allMessagesOfType)) {
+        List<Message> messages = messageRepository.findAllMessagesByTypeOrderedByDate(messageType);
+        if (!CollectionUtils.isEmpty(messages)) {
             MessageTypeProcessor messageTypeProcessor = messageProcessorsByType.get(messageType);
             if (messageTypeProcessor == null) {
                 throw new IllegalArgumentException("No message type processor found in application context for message type: "
-                        + messageType + ", there are " + allMessagesOfType.size() + " message(s) in the queue");
+                        + messageType + ", there are " + messages.size() + " message(s) in the queue");
             }
-            allMessagesOfType.forEach(messageTypeProcessor::processMessage);
+
+            processMessages(messageTypeProcessor, messages);
+        }
+    }
+
+    private void processMessages(MessageTypeProcessor messageTypeProcessor, List<Message> messages) {
+        log.info("Processing {} {} of type {}", messages.size(), messages.size() == 1 ? "message" : "messages", messageTypeProcessor.supportsMessageType());
```

Comment:
Can we have `messages.size() == 1 ? "message" : "messages"` either out as a function, or simply replaced with `message(s)`, I'd be happy with either. You've also done the same in the log message below too.

Reply:
Updated to message(s)